src/actions/websocket.js

```diff
@@ -388,7 +400,7 @@ function handleUserAddedEvent(msg) {
         }, getState);
 
         if (msg.broadcast.channel_id === currentChannelId) {
-            dispatch(getChannelStats(teamId, currentChannelId));
+            dispatch(getChannelStats(currentChannelId));
```

Comment:
based on the function signature, the code before `getChannelStats(teamId, currentChannelId)` was a bug. However I would like to get input from more experienced developers.

Reply:
Yes, that was a bug, clearly, I think is the correct behavior to change it here :)